server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const io = require('socket.io')(3000);
+
+let cnt = 0;
+let counter = 0;
+let letter = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'];
+
+io.on('connection', (socket) => {
```

Comment:
Check out this function, it's what's causing that error to be thrown when you start your server.